skimage/data/__init__.py

```diff
@@ -62,7 +62,12 @@
 skimage_distribution_dir = osp.join(legacy_data_dir, '..')
 
 try:
-    from pooch.utils import file_hash
+    try:
+        # Pooch >= 1.5.0
+        from pooch.hashes import file_hash
+    except ImportError:
+        # Pooch < 1.5.0
+        from pooch.utils import file_hash
```

Comment:
```suggestion
    from pooch import file_hash
```
I think we can just do this instead

I looked at the pooch source and it seems this function has always been available from the main pooch namespace: 
currently: https://github.com/fatiando/pooch/blob/master/pooch/__init__.py#L12
even in v0.1: https://github.com/fatiando/pooch/blob/v0.1/pooch/__init__.py#L5

Reply:
That's much more sensible.

Reply:
thank you both for looking into this correctly.